doc/source/security/data-at-rest-encryption.rst

```diff
@@ -166,6 +171,52 @@ after creating the encrypted tablespace. If you are using Master key encryption,
 The first time a key is retrieved from a `keyring`, the `keyring_vault`
 communicates with the Vault server to retrieve the key type and data.
 
+secret_mount_point_version information
+---------------------------------------
+
+Impemented in |Percona Server| 5.7.33-36, the ``secret_mount_point_version`` can be either a ``1``, ``2``, ``Auto`` or ``None``. 
```

Comment:
`None` is misleading here - this can be read as
```
secret_mount_point_version = None
```
Possible options are
```
secret_mount_point_version = 1 | 2 | AUTO
```
or there is no `secret_mount_point_version` parameter in the configuration file at all.

Reply:
Here and everywhere, the value of the `secret_mount_point_version` is `AUTO` (all upper case).

Reply:
Updated.